docs/guides/compose.md

```diff
@@ -0,0 +1,260 @@
+
+- `serverless logs` to fetch logs from **all functions across all services**
+- `serverless info` to view all services info
+- `serverless remove` to remove all services
+- `serverless outputs` to view all services outputs
+- `serverless refresh-outputs` to refresh outputs of all services
+
+For example, it is possible to tail logs for all functions at once:
+
+```bash
+$ serverless logs --tail
+
+service-a › users › START
+service-a › users › 2021-12-31 16:54:14  INFO  New user created
+service-a › users › END Duration: 13 ms ...
+service-b › subscriptions › START
+service-b › subscriptions › 2021-12-31 16:54:14  INFO  New subscription enabled
+service-b › subscriptions › END Duration: 7 ms ...
+
+    ⠴  service-a › logs › 2s
+    ⠦  service-a › logs › 2s
+
+```
+
+### Service-specific commands
+
+It is possible to run commands for a specific service only. For example to deploy only a specific service:
+
+```bash
+serverless deploy --service=service-a
+
+# Shortcut alternative
+serverless service-a:deploy
+```
+
+Or tail logs of a single function:
+
+```bash
+serverless logs --service=service-a --function=index
+
+# Shortcut alternative
+serverless service-a:logs --function=index
+```
+
+All Serverless Framework commands are supported **only via service-specific commands**, including custom commands from plugins, for example:
+
+```bash
+serverless service-a:offline
+```
+
+### Service-specific commands when using parameters
+
+The `serverless service-a:deploy` command is the equivalent of running `serverless deploy` in service-a's directory. Both can be used.
+
+However, if "service-a" uses `${param:xxx}` to reference parameters injected by `serverless-compose.yml`, then `serverless service-a:deploy` must be used. Indeed, `${param:xxx}` cannot be resolved outside of Serverless Framework Compose.
+
+In these cases, you must run all commands from the root: `serverless service-a:deploy`.
+
+## Configuration
+
+The following variables are supported in `serverless-compose.yml`:
+
+- [`${sls:stage}`](../providers/aws/guide/variables.md#referencing-serverless-core-variables)
+- [`${env:xxx}`](../providers/aws/guide/variables.md#referencing-environment-variables)
+
+### Differences with `serverless.yml`
+
+The `serverless-compose.yml` and `serverless.yml` files have different syntaxes and features.
+
+Unless documented here, expect `serverless.yml` features to not be supported in `serverless-compose.yml`. For example, it is not possible to include plugins or use most `serverless.yml` variables (like `${self:`, `${opt:`, etc.) inside `serverless-compose.yml`.
+
+You can [open feature requests](https://github.com/serverless/compose) if you need features that aren't supported in `serverless-compose.yml`.
+
+## Refreshing outputs
+
+The outputs of a service are stored locally to accelerate deployments. If a colleague deployed changes that changed the outputs of a service, you can refresh your local state via the `refresh-outputs` command:
```

Comment:
I think this is a misleading sentence. It's not stored locally to accelerate deployments (because we don't do any acceleration based on the outputs from local state at the moment), but rather to allow running e.g. component-specific commands without having to resolve the outputs each time.

Reply:
Good point!

"allow running e.g. component-specific commands without having to resolve the outputs each time." that's the part that I shortened to "accelerate" (because we don't fetch remote output) but I do see now how the sentence actually says something else. I reworded that!